.github/workflows/integration-test.yml

```
@@ -196,6 +196,9 @@ jobs:
       - name: Install Docker and Colima on MacOS
         if: runner.os=='macOS'
         run: |
+          export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=1
+          brew update
+          brew reinstall qemu
```

```suggestion
          # Workaround for https://github.com/actions/runner-images/issues/8104
          brew remove --ignore-dependencies qemu
          curl -o ./qemu.rb https://raw.githubusercontent.com/Homebrew/homebrew-core/dc0669eca9479e9eeb495397ba3a7480aaa45c2e/Formula/qemu.rb
          brew install ./qemu.rb
```

Maybe try this (as suggested in the thread)